packages/react/package.json

```diff
@@ -26,6 +26,9 @@
     "react": "15.x || 16.x",
     "react-dom": "15.x || 16.x"
   },
+  "optionalDependencies": {
+    "redux": ">=1.0.0"
```

Comment:
On second thought, maybe we don't want to do this. Having an optionDependency means that it will always been installed unless `--no-optional` is passed in. @kamilogorek what do you think?

Reply:
tbh I think we could completely drop it. You won't use it unless you already have redux in your project anyway.

Reply:
Would we just not use the types then in this case?

Reply:
I think we would need it in `devDependencies` at least, as the tests use redux

Reply:
yeah include it in `devDependencies`